Objects/codeobject.c

```diff
@@ -1151,46 +1140,39 @@ _PyCode_SetExtra(PyObject *code, Py_ssize_t index, void *extra)
 PyObject *
 _PyCode_GetVarnames(PyCodeObject *co)
```

Comment:
> This no longer gets the `varnames`, it makes it.

The functionality hasn't changed, it just doesn't cache its result anymore. I'd rather just leave the name alone, since it's quite clear as-is.

> Also, why this change?

These three members are hardly used, and recomputing them is not expensive (in time *or* space). I also don't think they're worth the additional cache pressure they likely place on actual bytecode now.